website/docs/visual-testing/integrate-with-percy.md

```diff
@@ -0,0 +1,144 @@
+---
+id: integrate-with-percy
+title: Run visual tests on your web apps with Percy
```

Comment:
instead of:
```
Run visual tests on your web apps with Percy
Run visual tests on your mobile apps with App Percy
```
Let's organize the content as:
```
Integrate with Percy
- For Web Apps
- For Mobile Apps
```

Reply:
Done. I have incorporated the suggested comments.